src/profile-logic/import/instruments/instruments.js

```diff
@@ -0,0 +1,749 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Main file for the changes.

Reply:
Since this is the root file of this `instruments` module, it would be better to rename this file to `index.js`, that way you'll be able to import this file like this: `import { something } from 'profile-logic/import/instruments';`

(Right now we need to write the instruments twice: `import { something } from 'profile-logic/import/instruments/instruments';`)